server/common/utils/helpers.js

```diff
@@ -64,7 +65,8 @@ const responseWithList = (list, userId) => {
   };
 
   if (cohortId) {
-    listToSend.cohortId = cohortId;
+    listToSend.cohortId =
+      typeof cohortId === 'string' ? cohortId : cohortId._id;
```

Comment:
can't we just always send id as a string ??

`listToSend.cohortId = cohortId.toString();`

Reply:
It can't be done in the way you proposed because once cohortId is a string, second time is an object (when you populate cohort data). To use this helper in both situation you have to check cohortId type if it is a string you can assign it to listToSend object directly if not you have to get _id from cohortId object.

Reply:
I think, in this case this should no longer be named `cohortId` when it is truly the `cohort` object but under confusing name. Please try to avoid that situation, maybe send the `cohort` and `cohortId` separately?